requirements.txt

```diff
@@ -1,8 +1,8 @@
 # SPDX-FileCopyrightText: 2024 Ledger SAS
+# SPDX-FileCopyrightText: 2025 H2Lab
 #
 # SPDX-License-Identifier: Apache-2.0
 
-meson >= 1.3.0
-dunamai >= 1.17.0
+meson
```

Comment:
mmm... not defining minimal version ? i.e. considering a host on which an old meson (<1.4) is installed, this line has no impact, whlie we may enforce minimal though ?